Modelica/Blocks/Sources.mo

```diff
@@ -593,12 +605,18 @@ and that the parameter <code>startTime</code> is omitted since the voltage can b
     parameter Real amplitude=1 "Amplitude of sine wave"
     annotation(Dialog(groupImage="modelica://Modelica/Resources/Images/Blocks/Sources/Sinc.png"));
     parameter SI.Frequency f(start=1) "Frequency of sine wave";
+    parameter Boolean continuous = false "Make output smooth by starting at offset + amplitude rather than just offset";
```

Comment:
It's still not infinitely differentiable:
```suggestion
    parameter Boolean continuous = false "Make output (continuously) differentiable by starting at offset + amplitude rather than just offset";
```

I guess the name `continuous` still makes sense to get similarity with the other models.